pandas/tests/extension/test_common.py

```diff
@@ -79,3 +79,9 @@ def test_astype_no_copy():
 def test_is_extension_array_dtype(dtype):
     assert isinstance(dtype, dtypes.ExtensionDtype)
     assert is_extension_array_dtype(dtype)
+
+
+@pytest.mark.parametrize("columns, dtype", [(["a"], "string")])
+def test_construct_empty_dataframe_with_string_dtype(columns, dtype):
```

Comment:
pls use the fixtures so this is generically tested. move to pandas/tests/extension/base/constructor.py (as a new test). sorry gave you the wrong path before.